### PR TITLE
Add image rotation

### PR DESCRIPTION
**Pull request by mklein-de**

### CODE REVIEW

File: src/image-viewer.cpp [hunk 1]
```diff
@@ -188,21 +189,40 @@ void ImageViewer::addTool(QWidget *tool) {
...
+}
+
+qreal ImageViewer::rotationRadians() const {
+    auto p10 = m_view->transform().map(QPointF(1., 0.));
+    return std::atan2(p10.y(), p10.x());
+}
+
+void ImageViewer::setRotation(qreal angle) {
+    m_view->rotate(angle - rotation());
+}
```
Comment (palacaze) [inline src/image-viewer.cpp:203]: This needs a
```cpp
    if (m_fit)
        zoomFit();
```

**Comment by palacaze** [comment]
I merged #5, can you rebase over the main branch so that I can merge this one too after the small additional change?

**Comment by mklein-de** [comment]
> I merged #5, can you rebase over the main branch so that I can merge this one too after the small additional change?

Done, thanks for reviewing and merging!

**Comment by palacaze** [comment]
Great, thanks!

**Pull request merged**